tests/components/calendar/test_init.py

```diff
@@ -419,35 +424,25 @@ async def test_list_events_service(
         blocking=True,
         return_response=True,
     )
-    assert response == {
-        "events": [
-            {
-                "start": "2023-06-22T05:00:00-06:00",
-                "end": "2023-06-22T06:00:00-06:00",
-                "summary": "Future Event",
-                "description": "Future Description",
-                "location": "Future Location",
-            }
-        ]
-    }
+    assert response == snapshot
 
 
 @pytest.mark.parametrize(
-    ("entity", "duration", "expected_events"),
+    ("entity", "duration"),
     [
         # Calendar 1 has an hour long event starting in 30 minutes. No events in the
         # next 15 minutes, but it shows up an hour from now.
-        ("calendar.calendar_1", "00:15:00", []),
-        ("calendar.calendar_1", "01:00:00", ["Future Event"]),
+        ("calendar.calendar_1", "00:15:00"),
```

Comment:
This is a good example of a parametrize where I think comparing against an empty data structure makes sense :)

Reply:
I especially want the snapshot here because the data structure will change between the two service responses.
The response payload gets directly accessed `events = response["events"]`. 

Without a snapshot I would have to introduce another test which handles the different payload structure. This verbosity is exactly what I want to avoid.